packages/integration-tests/tests/bridge_erc20.ts

```diff
@@ -359,6 +359,7 @@ describe('Bridging ERC20 tokens', async function () {
 
   describe('Bridge ERC20 from Fuel', async () => {
     const NUM_TOKENS = 10000000000000000000n;
+    const largeRateLimit = `30`;
```

Comment:
I see integration tests, but are there unit tests?

Reply:
Unless we are referring to different things, they do: https://github.com/FuelLabs/fuel-bridge/blob/6030a40ce9c58a533c09f73e837f85ab4784ef58/packages/solidity-contracts/test/behaviors/erc20GatewayV4.behavior.test.ts#L935